google-cloud-storage/src/main/resources/META-INF/native-image/com/google/cloud/storage/reflect-config.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "name":"org.apache.commons.logging.LogFactory",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name":"org.apache.commons.logging.impl.Jdk14Logger",
+    "methods":[{"name":"<init>","parameterTypes":["java.lang.String"] }]}
+,
+  {
+    "name":"org.apache.commons.logging.impl.LogFactoryImpl",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "methods":[{"name":"<init>","parameterTypes":[] }]}
+,
+{
+  "name":"com.google.cloud.storage.BlobInfo$ImmutableEmptyMap",
+  "methods":[{"name":"<init>","parameterTypes":[] }]}
+
+,
+{
+  "name":"com.google.cloud.storage.conformance.retry.TestBench$RetryTestResource",
+  "allDeclaredFields":true,
+  "methods":[{"name":"<init>","parameterTypes":[] }]}
```

Comment:
This class is test only. I don't think we'd want it in a manifest that ends up in our released jar. Is there anyway to move this to a sibling file under `src/test/resources/...`?

Reply:
Good catch, let's move this to src/test/resources to prevent it from ending up in the released jar.